yang/frr-ospfd.yang

```diff
@@ -0,0 +1,1000 @@
+        }
+      }
+
+      leaf node-msd {
+        type uint8 {
+          range "1..16";
+        }
+        description
+          "Maximum Stack Depth for this router.";
+      }
+
+      leaf on {
+        type boolean;
+        description
+          "Enable Segment Routing.";
+      }
+
+      list indices {
+        key "index";
+        leaf index {
+          type uint16;
+          description
+            "SID index for this prefix.";
+        }
+
+        leaf prefix {
+          type inet:ipv4-prefix;
+          description
+            "Prefix SID.";
+        }
+
+        leaf no-php-flag {
+          type boolean;
+          description
+            "Don't request Penultimate Hop Popping (PHP).";
+        }
+      }
+    }
+  }
+
+  grouping ospf-list-group {
+    list neighbor {
+      key "ip";
+      description
+        "Neighbor list.";
+      leaf ip {
+        type inet:ipv4-address;
+        description
+          "Neighbor IP address.";
+      }
+
+      leaf priority {
+        type uint8;
+        description
+          "Neighbor Priority.";
+      }
+
+      leaf poll-interval {
+        type uint16 {
+          range "1..65535";
+        }
+        units "seconds";
+        description
+          "Dead Neighbor Polling interval.";
+      }
+    }
+
+    list network {
+      key "prefix";
+      description
+        "Enable routing on list of IP network.";
+      leaf prefix {
+        type inet:ipv4-prefix;
+        description
+          "Prefix on which routing needs to be enabled.";
+      }
+
+      leaf area {
+        type ospf-area-id;
+        description
+          "Area ID for this network.";
+      }
+    }
+
+    list passive-interface {
+      key "interface";
+      description
+        "Suppress routing updates on list interface.";
+      leaf interface {
+        type frr-interface:interface-ref {
+        }
+        description
+          "Suppress routing updates on an interface.";
+      }
+
+      leaf address {
+        type inet:ipv4-address;
+        description
+          "Interface address.";
+      }
+    }
+  }
+
+  grouping pce-group {
+    container pce {
+      description
+        "PCE Router Information.";
+      leaf address {
+        type inet:ipv4-address;
+        description
+          "IP address of the PCE.";
+      }
+
+      list domain {
+        key "as";
+        leaf as {
+          type uint16;
+          description
+            "AS number where the PCE as visibilities for path computation.";
+        }
+      }
+
+      list neighbor {
+        key "as";
+        leaf as {
+          type uint16;
+          description
+            "AS number of PCE neighbors.";
+        }
+      }
+
+      leaf flag {
+        type uint32;
+        description
+          "Capabilities of the PCE for path computation.";
+      }
+
+      leaf scope {
+        type uint32;
+        description
+          "Path scope visibilities of the PCE for path computation.";
+      }
+    }
+  }
```

Comment:
Not sure if we need to keep this. PCE advertisement in Router Information is not used. I envisage to remove it from the code.

Reply:
I would agree -- there's no real reason for this, as this part of PCE really isn't used any longer afaik.

Reply:
As per discussion, I have removed the pce group container from yang.